automation/check-patch.e2e-k8s.sh

```diff
@@ -31,7 +31,13 @@ main() {
     make cluster-up
     trap teardown EXIT SIGINT SIGTERM SIGSTOP
     make cluster-sync
-    make E2E_TEST_TIMEOUT=1h E2E_TEST_ARGS="-noColor" E2E_TEST_SUITE_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml" test-e2e-handler
+
+    if [ $NMSTATE_PARALLEL_ROLLOUT == "true" ]; then
+      export E2E_TEST_SUITE_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml -ginkgo.skip='user-guide'"
+    else
+      export E2E_TEST_SUITE_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml -ginkgo.skip='parallel'"
+    fi
```

Comment:
Let's concantenat E2E_TEST_SUITE_ARGS so we don't repeat ourselves.

Reply:
Updated